src/_data/LiveData.js

```diff
@@ -84,13 +84,19 @@ const initAuthorized = async (authData, store) => {
             }
         });
 
-
+    const susbcribeWatchlist = async (symbol) => {
```

Comment:
And this method have nothing to do with watchlist, I think an inline closure would do